### ISSUE TITLE
ctrl+a selects all instead of opening prompt

### ISSUE THREAD

**Issue by matt-cook** [zebradog/kiosk#87]
reported as happening on a Chromebox.

![](https://api.bountysource.com/badge/issue?issue_id=30724397)

**Comment by jbyauk** [comment]
I am having this issue as well. Ctrl+A does not open the settings dialog on an unmanaged Asus Chromebox M004U with the Kiosk extension running in kiosk mode from boot.

**Comment by matt-cook** [comment]
Thanks for the report. Do you have "Local Administration" and "Disable Text Selection" options turned on? Leaving both of these options off will behave as described.

I'm also curious: How are you running kiosk mode from boot without a managed Chromebox? Are you using it in Developer mode?

**Comment by jbyauk** [comment]
Ah, you're right. I had forgotten that I disabled "Local Administration" initially on setup.

To run the Chromebox in kiosk mode unmanaged, I followed the steps in [this post](https://productforums.google.com/d/msg/chromebook-central/XTqYPQfZmNc/v52bI80z4ygJ) on the Chromebook Google product forums.  I was unable to get Chrome Sign Builder running in this configuration without the required remote settings in place, but the Kiosk extension works perfectly since it has Local Administration.
